packages/state-manager/src/sagas/communities/communities.slice.ts

```diff
@@ -57,9 +48,34 @@ export const communitiesSlice = createSlice({
     clearInvitationCodes: state => {
       state.invitationCodes = []
     },
-    saveCommunityMetadata: (state, _action: PayloadAction<CommunityMetadata>) => state,
-    savePSK: (state, action: PayloadAction<string>) => {
-      state.psk = action.payload
+    /**
+     * Migrate data in this store. This is necessary because we persist the
+     * Redux data to disk (it's not reset on each app start). This function is
+     * meant to be called once the store has been rehydrated from storage.
+     *
+     * TODO: This type of thing might be better suited in a saga where we can
+     * organize complicated migration code easier.
+     */
+    migrate: state => {
```

Comment:
Have you considered using redux-persist migration mechanism?

Reply:
I didn't know that existed. I'm looking into it now. I think it might actually be a cleaner way to do this. Thanks!